localgov_blogs.module

```diff
@@ -144,8 +145,11 @@ function localgov_blogs_field_widget_complete_form_alter(&$field_widget_complete
     // If there are no blog channels display a warning.
     if (count($options) == 1 && isset($options['_none'])) {
       $create_channel_path = 'localgov_blog_channel';
-      if (\Drupal::hasService('domain_group_resolver') && !is_null(\Drupal::service('domain_group_resolver')->getCurrentRouteGroupDomain())) {
-        $create_channel_path = 'group_node:' . $create_channel_path;
+      if (\Drupal::moduleHandler()->moduleExists('localgov_microsites_group')) {
+        $group = localgov_microsites_group_get_by_context();
+        if ($group instanceof GroupInterface) {
```

Comment:
This could be `if ($group)` as `localgov_microsites_group_get_by_context()` only returns NULL or GroupInterface.